src/rbac/schema.js

```diff
@@ -1,10 +1,11 @@
 const Joi = require('joi');
 
 const roleName = Joi.string().alphanum().max(500);
+const roleNames = Joi.array().items(roleName);
```

Comment:
should this have `single()`, `max(...)`?

also are we losing the requirement of at least one role here?